rules-tests/Php82/Rector/New_/FilesystemIteratorSkipDots/Fixture/maintain_default_value.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
please rename fixture file to `skip_with_default_value.php.inc`

Reply:
I renamed the file